apps/frontend/src/pages/project/project.tsx

```diff
@@ -31,34 +42,49 @@ const Project = (): JSX.Element => {
 
 	const isRejected = projectStatus === DataStatus.REJECTED;
 
-	const hasProject = Boolean(project);
+	const hasProject = project !== null;
 
 	if (isRejected) {
 		return <NotFound />;
 	}
 
 	return (
 		<PageLayout isLoading={isLoading}>
-			<div className={styles["breadcrumb-container"]}>
-				{hasProject && (
-					<Breadcrumbs
-						items={[
-							{ href: AppRoute.ROOT, label: "Projects" },
-							{ label: (project as ProjectGetAllItemResponseDto).name },
-						]}
-					/>
-				)}
-			</div>
+			{hasProject && (
+				<>
+					<div className={styles["breadcrumb-container"]}>
+						<Breadcrumbs
+							items={[
+								{ href: AppRoute.ROOT, label: "Projects" },
+								{ label: project.name },
+							]}
+						/>
+					</div>
+
+					<div className={styles["project-layout"]}>
+						<h1 className={styles["title"]}>{project.name}</h1>
+						<div className={styles["project-description-layout"]}>
+							<h3 className={styles["project-description-title"]}>
+								Description
+							</h3>
+							<p className={styles["project-description"]}>
+								{project.description}
+							</p>
+						</div>
+						<Button
+							label="Setup analytics"
+							onClick={onSetupAnalyticsModalOpen}
+						/>
+					</div>
```

Comment:
Is it looks okay? or it is better to rid of two extra identation by returning empty page layout in case there is no project (`!hasProject`)

Reply:
I would say the second option. And from the logic we have there shouldn't be a case where it's null